lib/is_octagonal.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+/**
+  * Determines whether an integer is 'octagonal'.
+  * @param {Number} an integer x to test
+  * @return {Boolean} Whether or not the x is octagonal.
+  * @module number-theory
+  * @author Kelly Innes
+  */
+
+module.exports = function isOctagonal(n) {
+  return (((Math.sqrt((3 * n) + 1)) + 1) / 3) % 1 == 0;
```

Comment:
Same here, add a comment to where you got the formula (proof would be great). Ditto for the rest of these. Mostly I'd like to make reading the source educational :)